### ISSUE TITLE
Links opened from other apps do not load by themselves

### ISSUE THREAD

**Issue by Cwpute** [scoute-dich/browser#507]
As referenced in issue #506, new blank tabs make the browser crash. Weirdly enough though, tabs opened from links in other apps do not make the browser crash, though they require the user to manually reload the page for it to ctually load at all.

Again, as far as i know, this behavior only occured since last update (currently using FOSS Browser v7.4.3) but it might have to do with the new design update that rolled out recently.
I'm using Android Webview Version 89.0.4389.105, because later versions made any webview based apps on my phone crash.
I'm on a Huawei Honor BLN-L21, running Android 7.0.

I'll be switching to the last version prior to the design update until this issue is fixed.

Thanks for your time !

**Comment by scoute-dich** [comment]
The test-apk in issue #506 could also solve this. If you have the time, you can test it.

**Comment by tusch001** [comment]
The latest apk from #506:

I can't comment on the crashes from #506 (I don't get them - sorry no logcat), but external links still result in a blank page and the site has to be reloaded manually.

Also nothing in the logcat (I used the external link from #497 ):
```
---- 23.05.2021 2:03:24 nachm. ----

05-23 14:02:40.690  1249  4531 I PackageManager: Setting last chosen activity de.baumann.browser_test/de.baumann.browser.activity.BrowserActivity for user 0:

05-23 14:02:40.692  1249  5117 I ActivityManager: START u0 {act=android.intent.action.VIEW dat=https://apps.evozi.com/... flg=0x3000000 cmp=de.baumann.browser_test/de.baumann.browser.activity.BrowserActivity (has extras)} from uid 10134 on display 0

---- 23.05.2021 2:03:24 nachm. ----
```

Android 6.0.1

Is there anything else I can do to help you?

**Comment by Cwpute** [comment]
Test-apk from issue #506 didn't resolve the issue. 

I couldn't record any logcat either, as it seems i do either require a usb connection to a computer, or that my phone is rooted - which it isn't. Maybe there is a specific app that would let me without these requirements ? Scoop (https://github.com/TacoTheDank/Scoop) semed like it allowed me to, but the logcat didn't show anything regarding FOSS Browser after i stopped recording.

**Comment by tusch001** [comment]
@Cwpute:
I don't think it is possible to get logcats without a rooted phone.

**Comment by scoute-dich** [comment]
[app-debug.apk.zip](https://github.com/scoute-dich/browser/files/6600770/app-debug.apk.zip)
I made some changes in the code. Since I can not reproduce this issue, I'll need you to test. Thanks in advance.

_P.S.: Tell me also, which Android version you are using while testing._

**Comment by scoute-dich** [comment]
I just released a new version (7.5) if this bug still is not solved, please report it here. Otherwise, I'll close this issue.

**Comment by Cwpute** [comment]
> I just released a new version (7.5) if this bug still is not solved, please report it here. Otherwise, please close this issue (Or I'll do that in the near future).

The smartphone i was using was lost in an unfortunate boat accident, i won't be able to test out the new version. My excuses for the long awaited reply, i was simply unable to access internet without it, and when i did manage to, i had other more pressing stuff to attend to. I'll be sure to report if i find any other similar bug if i happen to run into on !
Thanks for your time and support on the matter !
